js/store.js

```diff
@@ -11,6 +10,91 @@ const store = {
       }
     }
     browser.runtime.onMessage.addListener((m) => store.messageHandler(m));
+
+    // get Disconnect Entity List from shavar-prod-lists submodule
+    let whiteList;
```

Comment:
Yea separate function would be a little cleaner. Feel free to put hard coded contants at the top of the file too with long var names. Like `const WHITELIST_BLAH_SOMETHING_URL = ....`